data/passim.service.in

```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=A local caching server
 Documentation=https://github.com/hughsie/passim
-After=dbus.service
+After=dbus.service avahi-daemon.service
```

Comment:
Do we still need dbus.service?

Reply:
Actually, I think it was never needed. The service is of `Type=dbus` which according to [system.service(5)](https://man7.org/linux/man-pages/man5/systemd.service.5.html#AUTOMATIC_DEPENDENCIES) automatically adds "dependencies of type Requires= and After= on dbus.socket".